### ISSUE TITLE
space-before-keyword matches getters

### ISSUE THREAD

**Issue by remcohaszing** [eslint/eslint#3854]
After updating from ESLint 1.4.3 to 1.5.0 I noticed the `space-before-keywords` rule with configuration `always` matches getters and setters.

```js
return {
  get prop() {  <-- reports: Missing space before keyword "(".
    return prop;
  }
};
```

This conflicts with `space-before-function-paren` set to `never`.

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by mmrko** [comment]
Thanks for reporting this @remcohaszing. Unfortunately I won't have time look into this until after next week. Should be a relatively easy fix however :)

**Comment by mysticatea** [comment]
Thanks for the report.
I can reproduce this in the online demo.
I will try to fix this.

![image](https://cloud.githubusercontent.com/assets/1937871/9981373/31f66526-5ff6-11e5-9ad9-04ce2b9b1b08.png)

**Comment by gyandeeps** [comment]
@mysticatea Can you share on how you are going to solve this?
This would help everyone to understand what the change is going to be so that everyone can weigh in on it. Situation like these some times produce breaking changes.

**Comment by mysticatea** [comment]
@gyandeeps Oh, OK. I realized that `get` is a keyword right now.
I had been making `space-before-keywords` ignoring getters and setters. But actually, spacing which is between `}` and `get` (or `set`) should be handled by this rule.

**Comment by mysticatea** [comment]
In fact, `get` and `set` are not keywords: http://www.ecma-international.org/ecma-262/6.0/#sec-keywords

Hmm, `class A { foo(){} bar(){} }` spacing which is between `foo(){}` and `bar` is not handled by this rules. I rethought `space-before-keywords` should ignore getters, setters, and methods merely.
